nifi-nar-bundles/nifi-standard-services/nifi-hwx-schema-registry-bundle/nifi-hwx-schema-registry-nar/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Copyright 2016 Hortoworks, Inc. All rights reserved. 
```

Comment:
this is not correct.  This is apache nifi code.  This header would only make sense if we're copying source from hortonworks repos.